### ISSUE TITLE
Store Robberies

### ISSUE THREAD

**Issue by Juhux** [echoroleplay/feedback#378]
Suggestion Description:
Make stores work in a different way similar to Houses with a better risk/reward system whilst avoiding hostage situations. Hear me out. For example, the safe would have a hack and that hack if successful rewards you with something similar to house safes (1.5-2k) + the ability to maybe shoplift/loot various spots inside a store that might give you an item for crime progression. If the hack is successful, cops wouldn't get notified and if the robber fails, they do, just like houses. In that case, the robber chooses to either flee or continue looting said spots in the store. All that with a similar queue system for houses and similar cooldowns.

Reasoning:
Stores right now are...worthless, they aren't worth the risk for a couple notes. Making them like houses makes it so that their risk/reward ratio makes sense. You either pull off the hack and make around 1-2k or you don't and you get cops and/or have to dip just like in a house robbery. I am pretty sure this would benefit both criminals and cops all while avoiding the main problem of stores which was constant hostage situations. I believe the frequency of hostage situations really needed to be looked at and it was, but that doesn't mean that stores should be useless and not worth doing at all. 

Additional context
Stores were fun mayne.

**Comment by claymstehling** [comment]
Agreed, bringing back that interaction for police would be amazing as well. I miss a good ole store robbery

**Comment by damightylamb** [comment]
Pd hated that old interaction of the old stores

**Comment by claymstehling** [comment]
Wait really? I loved it from my end

**Comment by FlawwsX** [comment]
> Wait really? I loved it from my end

From what we have read from majority of higher up cops, they disliked it.

**Comment by Juhux** [comment]
> > Wait really? I loved it from my end
> 
> From what we have read from majority of higher up cops, they disliked it.

I thought they disliked the hostage aspect of it because they were having to break off other calls to clear hostages. I don't believe they would dislike stores being a hostage-less call just like drug sales/houses (which is what stores are right now) but it can be improved is the whole point of the suggestion. Hence why I said without going back to old ways/hostages.

**Comment by claymstehling** [comment]
NGL I enjoyed responding to them. I had some very good scenes come from the old way
